android/src/com/mapswithme/maps/widget/placepage/PlacePageView.java

```diff
@@ -380,6 +382,9 @@ protected void onFinishInflate()
     mOperator = findViewById(R.id.ll__place_operator);
     mOperator.setOnClickListener(this);
     mTvOperator = findViewById(R.id.tv__place_operator);
+    mLevel = findViewById(R.id.ll__place_level);
+    mLevel.setOnClickListener(this);
```

Comment:
Does label have any onclick listener? As far as I can see only long clicks are handled.

Reply:
You are right, it's not needed. I just removed it. Thanks!